packages/data-store/src/entities/contact/PartyTypeEntity.ts

```diff
@@ -11,21 +11,21 @@ export class PartyTypeEntity {
   @PrimaryGeneratedColumn('uuid')
   id!: string
 
-  @Column('simple-enum', { name: 'type', enum: PartyTypeType, nullable: false, unique: false })
+  @Column('simple-enum', { name: 'type', enum: PartyTypeType, nullable: false })
   type!: PartyTypeType
 
-  @Column('simple-enum', { name: 'origin', enum: PartyOrigin, default: 'EXTERNAL', nullable: false, unique: false })
+  @Column('simple-enum', { name: 'party_origin', enum: PartyOrigin, default: 'EXTERNAL', nullable: false, unique: false })
```

Comment:
I have to differentiate between part origin and identity origin because typeorm doesn't like to join tables with the same column name